pl.lproj/Localizable.strings

```diff
@@ -0,0 +1,160 @@
+
+"WELCOME_ADD_PRESETS" = "Dodaj Stacje Początkowe";
+"WELCOME_NO_THANKS" = "Nie, dziękuję";
+
+"GENERATED_COLLECTION_TITLE" = "Moja Kolekcja";
+
+"UPSELL_COPY" = "Dzięki za korzystanie z Broadcasts!\n\nJeżeli chcesz, możesz odblokować zaawansowane funkcje zakupem w aplikacji.\n\nJeżeli wolisz, możesz po prostu edytować lub korzystać dalej ze stacji początkowych!";
+"UPSELL_BUY" = "Kup (%@)";
+"UPSELL_NOT_AVAILABLE" = "Zakup Nie Dostępny";
+"UPSELL_RESTORE" = "Odnów Zakup";
+"UPSELL_NO_THANKS" = "Nie, Dziękuję";
+
+"TOOLBAR_NEW_CATEGORY_TOOLTIP" = "Nowa Kolekcja";
+"TOOLBAR_ADD_STATION_TOOLTIP" = "Dodaj Stację";
+"TOOLBAR_SHOW_SOURCES_TOOLTIP" = "Pokaż Kolekcje";
+"TOOLBAR_TOGGLE_SIDEBAR_TOOLTIP" = "Przełącz Pasek Boczny";
+"TOOLBAR_TOGGLE_VIEW_TOOLTIP" = "Przełącz Widok";
+
+"IMAGE_WELL_REMOVE" = "Usuń";
+"IMAGE_WELL_PASTE" = "Wklej";
+"IMAGE_WELL_COPY" = "Kopiuj";
+
+"CARPLAY_EMPTY_LIBRARY" = "Brak Kolekcji";
+"CARPLAY_ALL_STATIONS" = "Wszystkie Stacje";
+
+"MAC_ADD_STATION_BUTTON" = "Dodaje Stację";
+"MAC_EDIT_STATION_BUTTON" = "Edytuj Stację";
+
+"MAC_ADD_COLLECTION_BUTTON" = "Dodaj Kolekcję";
+"MAC_EDIT_COLLECTION_BUTTON" = "Edytuj Kolekcję";
+
+"STATION_DIRECTORY_WINDOW_TITLE" = "Przeglądaj Stacje"
+"STATION_DIRECTORY_COUNTRIES_TITLE" = "Kraje";
+"STATION_DIRECTORY_STATIONS" = "Stacja %@";
+"STATION_DIRECTORY_STATIONS_PLURAL" = "Stacje %@";
+
+"STATION_DIRECTORY_NO_CONNECTION" = "Brak Połączenia";
+"STATION_DIRECTORY_CONNECTING" = "Ładuję…";
+
+"STATION_DIRECTORY_TOOLBAR_ADD_MANUALLY" = "Dodaj Ręcznie";
+"STATION_DIRECTORY_TOOLBAR_DONE" = "Gotowe";
+
+"STATION_DIRECTORY_COUNTRIES_SEARCH_PLACEHOLDER" = "Szukaj Kraj";
+"STATION_DIRECTORY_STATIONS_SEARCH_PLACEHOLDER" = "Szukaj Stację";
+
+"ADD_STATION_CONTEXT_ADD_STATION" = "Dodaj Ręcznie";
+"ADD_STATION_CONTEXT_SHOW_DIRECTORY" = "Dodaj z Katalogu";
+
+"LIBRARY_STATE_UPLOADING" = "Wysyłam…";
+"LIBRARY_STATE_DOWNLOADING" = "Pobieram…";
+"LIBRARY_STATE_PREPARING" = "Przygotowuję…";
+"LIBRARY_STATE_READY" = "Aktualny";
+"LIBRARY_STATE_MIGRATING" = "Migruję…";
+"LIBRARY_STATE_LOCALONLY" = "Tylko Lokalny";
+
+"TV_TAB_ALL_STATIONS" = "Wszystkie Stacje";
+"TV_TAB_RECENTS" = "Ostatnio Odtwarzane";
+"TV_TAB_COLLECTIONS" = "Kolekcje";
+
+"TV_NO_LIBRARY_MESSAGE" = "Biblioteka na Apple TV zsynchronizuje się po utworzeniu jej na iPhonie, iPadzie lub Macu";
+
+"WATCH_NOW_PLAYING_TITLE" = "Teraz Odtwarzane";
+"WATCH_SOURCE_RECENTS" = "Ostatnie";
+"WATCH_SOURCE_COLLECTIONS" = "Kolekcje";
+
+"WATCH_LIBRARY_SYNC_IN_PROGRESS" = "Synchronizuję…";
+
+"ALERT_IMPORT" = "Importuj";
+"IMPORT_TITLE" = "Importować \"%@\"?";
+"IMPORT_BODY" = "Ta udostępniana kolekcja zawiera %ld stacji.";
+
+"MOVE_DIALOG_TITLE" = "Przenieść \"%@\"";
+"MOVE_DIALOG_BODY" = "Wybierz kolekcję, do której chcesz przenieść tą stację.";
```

Comment:
tą stację → tę stację (grammar)

Reply:
You could actually simplify this from:

`"IMPORT_BODY" = "Ta udostępniana kolekcja zawiera %ld stacji.";`

to

`"IMPORT_BODY" = "Udostępniana kolekcja zawiera %ld stacji.";`